src/Illuminate/Foundation/Console/EventListCommand.php

```diff
@@ -97,4 +101,55 @@ protected function filteringByEvent()
     {
         return ! empty($this->option('event'));
     }
+
+    /**
+     * Adds the event/listeners on the dispatcher object to the given list.
+     *
+     * @param  array  $events
+     *
+     * @return array
+     */
+    protected function addListenersOnDispatcher(array $events)
+    {
+        foreach ($this->getRawListeners() as $event => $rawListeners) {
+            foreach ($rawListeners as $rawListener) {
+                if (is_string($rawListener)) {
+                    $events[$event][] = $rawListener;
+                } elseif ($rawListener instanceof Closure) {
+                    $events[$event][] = $this->stringifyClosure($rawListener);
+                }
+            }
+        }
+
+        return $events;
+    }
+
+    /**
+     * Creates a printable string version of a closure.
+     *
+     * @param  Closure  $rawListener
+     *
+     * @return string
+     */
+    protected function stringifyClosure(Closure $rawListener)
+    {
+        $reflection = new ReflectionFunction($rawListener);
+        $path = str_replace(base_path(), '', $reflection->getFileName() ?: '');
+
+        return 'Closure at: '.$path.':'.$reflection->getStartLine();
+    }
+
+    /**
+     * Gets the raw version of event listeners from dispatcher object.
+     *
+     * @return array
+     */
+    protected function getRawListeners()
+    {
+        try {
+            return $this->getLaravel()->make('events')->getRawListeners();
+        } catch (\Throwable $e) {
```

Comment:
Why?

Reply:
It was in the try/catch because I thought that it is possible that the container is not able to resolve the 'events' key and throws an exception.